x-pack/test/cloud_security_posture_functional/pages/compliance_dashboard.ts

```diff
@@ -66,5 +69,32 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     // describe('TODO - Cloud Dashboard', () => {
     //   it('todo - displays accurate summary compliance score', async () => {});
     // });
+
+    describe('Access with custom roles', async () => {
+      before(async () => {
+        await cspSecurity.createRoles();
+        await cspSecurity.createUsers();
+      });
+      after(async () => {
+        await cspSecurity.cleanRoles();
+        await cspSecurity.cleanUsers();
+      });
+
+      it('Access with valid user role', async () => {
+        await cspSecurity.logout();
```

Comment:
Should we include logout in the `after` callback?

Just a thought, do we need to logout if we clean the roles and users in the `after` callback?

Reply:
> Just a thought, do we need to logout if we clean the roles and users in the after callback?
When the test starts, we are by default logged in to `test_user`